pages/common/vim.md

```diff
@@ -1,32 +1,36 @@
 # vim
 
 > Vi IMproved, a programmer's text editor, providing several modes for different kinds of text manipulation.
-> Pressing `i` enters edit mode; the normal mode (accessed via `<Esc>`) doesn't allow regular text editing.
+> Pressing `i` enters edit mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
 
 - Open a file:
 
 `vim {{file}}`
 
 - Enter text editing mode (insert mode):
 
-`<Esc> i`
+`<Esc>i`
 
 - Copy ("yank") or cut ("delete") the current line (paste it with `P`):
 
-`<Esc> {{yy|dd}}`
+`<Esc>{{yy|dd}}`
 
 - Undo the last operation:
 
-`<Esc> u`
+`<Esc>u`
 
-- Search for a pattern in the file (press `n` to go to the next result):
+- Search for a pattern in the file:
 
-`<Esc> /{{search_pattern}} <Enter>`
+`<Esc>/{{search_pattern}}<Enter>   then   n (next), N (previous)`
 
-- Perform a regex substitution in the whole file (from the start, `1`, to the end, `$`):
+- Perform a regex substitution in the whole file:
 
-`<Esc> :1,$s/{{pattern}}/{{replacement}}/g <Enter>`
+`<Esc>:%s/{{pattern}}/{{replacement}}/g<Enter>`
```

Comment:
I understand your point, and agree to some extent; but we should keep in mind that tldr-pages is a didactic resource, not a cheatsheet of directly usable examples (that's more like [commandlinefu](http://www.commandlinefu.com/) territory). I don't think I can explain my motivations any better than I already have, so I'll just add that while I won't veto this change, I really, really would like us to keep `1,$` (and its corresponding note in the command description) for its didactic value.